src/api/appUsers.js

```diff
@@ -1,23 +1,69 @@
-import BaseAPI from './base';
-import http from '../utils/http';
+import { BaseAPI } from './base';
+import { http } from '../utils/http';
 
-export default class AppUsersAPI extends BaseAPI {
-  retrieve(userId, auth) {
-    const url = this.getFullURL('appUsers', userId);
+
+/**
+ * Init API properties
+ * @typedef InitProps
+ */
+
+/**
+ * @class AppUsersAPI
+ * @extends BaseAPI
+ */
+export class AppUsersAPI extends BaseAPI {
+
+  /**
+   * Initializes the conversation for a user
+   * @param  {InitProps} props
+   * @param  {AuthCredentials} auth
+   * @return {APIResponse}
+   */
+  init(props, auth) {
```

Comment:
I'm also not sure if I like auth being a parameter on individual methods. Makes mores sense to me that it be put in the `Smooch` constructor, and that you would instantiate separate Smooch instances when you wish to act on behalf of different auth roles.